src/snovault/util.py

```diff
@@ -56,16 +57,32 @@ def expand_path(request, obj, path):
         expand_path(request, value, remaining)
 
 
+def get_calculated_properties_from_paths(request, paths):
+    root = request.registry[ROOT]
+    calculated_fields = set()
+    item_types = {
+        root.collections.get(p.split('/')[1]).type_info.item_type
```

Comment:
don't like naked [1] here.  I think it's better to add a line of code parsing the split()